exercises/practice/gigasecond/test_gigasecond.c

```diff
@@ -1,6 +1,8 @@
 #include "test-framework/unity.h"
 #include "gigasecond.h"
 
+#define BUFFER_SIZE 20   // "YYYY-mm-dd HH:MM:SS"
```

Comment:
```suggestion
#define BUFFER_SIZE (strlen("YYYY-mm-dd HH:MM:SS") + 1)
```

Reply:
Alternatively: `#define BUFFER_SIZE sizeof("YYYY-mm-dd HH:MM:SS")`

Reply:
> Alternatively: `#define BUFFER_SIZE sizeof("YYYY-mm-dd HH:MM:SS")`

That's much better! I don't know why that never occurred to me.

Reply:
Should it be a `static size_t` instead of a #define so it only needs to be calculated once?

Reply:
It will be calculated at compile time even if it's a `#define`.